reference/configuration/tidb-server/configuration-file.md

```diff
@@ -16,18 +16,29 @@ The TiDB configuration file supports more options than command-line parameters.
 - Default value: `true`
 - It is recommended to set it to `false` if you need to create a large number of tables.
 
-### `oom-action`
-
-- Specifies the operation when out-of-memory occurs in TiDB.
-- Default value: `log`
-- The valid options are `log` and `cancel`. `log` only prints the log without actual processing. `cancel` cancels the operation and outputs the log.
-
 ### `mem-quota-query`
 
 - The maximum memory available for a single SQL statement.
 - Default value: `34359738368`
 - Requests that require more memory than this value are handled based on the behavior defined by `oom-action`.
 
+### `oom-use-tmp-storage`
+
++ Set to `true` to enable use of temporary disk for some executors when `mem-quota-query` is exceeded.
++ Default value:  `true`
++ Valid options: `true` or `false`
+
+### `tmp-storage-path`
+
++ The location of temporary disk storage using by some executors when `mem-quota-query` is exceeded.
++ Default value: `<TMPDIR>/tidb/tmp-storage`
+
+### `oom-action`
+
+- What to do when `mem-quota-query` is exceeded and can not be spilled over to disk any more.
+- Default value: `log`
+- The valid options are `log` and `cancel`. `log` only prints the log without actual processing. `cancel` cancels the operation and outputs the log.
```

Comment:
```suggestion
- Specifies what operation TiDB performs when a single SQL statement exceeds the memory quota and the temporary storage is disabled.
- Default value: `"log"`
- The valid options are `"log"` and `"cancel"`. When `oom-action="log"`,  it prints the log only. When `oom-action="cancel"`, it cancels the operation and outputs the log.
```

Reply:
Even the temporary storage is enabled, a query may alse exceed the memory quota because some operators cannot use the temporary disk.

Reply:
Thanks for clarifying. I update the comment.

Reply:
I suggest, 
Specifies what operation TiDB performs when a single SQL statement exceeds the memory quota specified by `mem-quota-query` and can not be spilled over to disk anymore.

It means the operator will try its best effort to spill into the temporary disk (If the `oom-use-tmp-storage` is enabled) when `mem-quota-query` is exceeded. After that, if `mem-quota-query` is still exceeded, then `oom-action` is triggered.

Reply:
Cool! This is much clearer. I have a minor edit:
> Specifies what operation TiDB performs when a single SQL statement exceeds the memory quota specified by `mem-quota-query` and cannot be spilled over to disk anymore.